src/components/wallet/PageTransactions.vue

```diff
@@ -197,6 +197,31 @@ export default {
           ]
         }
         return this.loadedTransactions
+      },
+      subscribeToMore: {
+        document: gql`
+          subscription($networkId: String!, $address: String!) {
+            userTransactionAddedV2(networkId: $networkId, address: $address) {
```

Comment:
Looking at `resolvers.js`, the code for both `userTransactionAdded` and `userTransactionAddedV2` are exactly the same.

What I would like to ask here is: we don't really need to use V2 to add this multiple subscription or we do? Please, I'd love to know

Reply:
I think it's a temporary solution, after switching to transaction abstraction we will replace every v2 in function names

Reply:
the returned schema is different. check schema.js